### DISCUSSION TITLE
Can we search anything and get the list of videos(videoId)

### DISCUSSION THREAD

**Discussion by Vedantmodi39** [PierfrancescoSoffritti/android-youtube-player#1015] [Q&A] [answered]
Can we search anything and get the list of videos(videoId)

**Comment by PierfrancescoSoffritti** [comment, marked as answer]
You need to use the YouTubeData API for that.

**Reply by Vedantmodi39** [reply]
But now it is deprecated that's why I asked

**Reply by PierfrancescoSoffritti** [reply]
It's not: [YouTube Data API](https://developers.google.com/youtube/v3/getting-started).

**Reply by Vedantmodi39** [reply]
Okay Got It. Youtube Player is deprecated.
